code/datums/diseases/advance/symptoms/stimulant.dm

```diff
@@ -1,38 +0,0 @@
-/*
-//////////////////////////////////////
-
-Stimulant //gotta go fast
-
-	Noticable.
-	Lowers resistance significantly.
-	Decreases stage speed moderately..
-	Decreases transmittablity tremendously.
-	Moderate Level.
-
-Bonus
-	The body generates Ephedrine.
-
-//////////////////////////////////////
-*/
-
-/datum/symptom/stimulant
-
-	name = "Stimulant"
-	stealth = -1
-	resistance = -3
-	stage_speed = -2
-	transmittable = -4
-	level = 3

```

Comment:
Maybe if it was simply split into two symptoms, as another alternative? Put the sanic-speed-gotta-go-fast in one, and the stun reduction in another. That'd certainly lead to virologists having to make important choices, simply due to the increased number of symptoms.

Non-antag viros have no need to give stun reduction to anyone, while heads and sec should have boosted footspeed. Antag viros, on the other hand, simply won't be able to make their ideal viruses.